src/v/cloud_storage/partition_manifest.cc

```diff
@@ -1136,6 +1161,78 @@ partition_manifest::find(model::offset o) const {
     return it;
 }
 
+partition_manifest::const_iterator partition_manifest::resolve_read(
+  const storage::log_reader_config& config,
```

Comment:
I can move it back into `remote_partition`. Since it's strictly a manifest operation this felt like a good place, but I don't really mind where this lives.

Reply:
Done.